Assets/TextAssets/TXT/TheMemories.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+
+public class TheMemories : MonoBehaviour 

```

Comment:
Once again, would recommend against using prepositions. To clarify (though I was also there working with you to write this class), is the intent that this stores all the memories? Or merely to test the current code? 

If it's to store all memories, it will likely need additional logic for how to add in multiple memories.

If it's just for testing, the class name should reflect that, e.g. `MemoryTest`